WcaOnRails/spec/models/solve_time_spec.rb

```diff
@@ -0,0 +1,48 @@
+def solve_time(centis)
```

Comment:
Note to myself: This file is missing a `require 'solve_time'` otherwise it can't be run using `bundle exec rspec spec/models/solve_time_spec.rb`